addons/mail/static/src/components/discuss_public_view/discuss_public_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.DiscussPublicView" owl="1">
+        <ThreadView t-if="discussPublicView.threadView"
+            composerAttachmentsDetailsMode="'card'"
+            hasComposer="true"
+            hasComposerThreadTyping="true"
+            hasSquashCloseMessages="true"
+            threadViewLocalId="discussPublicView.threadView.localId"
+        />
+        <WelcomeView t-if="discussPublicView.welcomeView"
+            welcomeViewLocalId="discussPublicView.welcomeView.localId"
+        />
+    </t>
```

Comment:
not a valid template, it should always have exactly one root element unconditionally

Reply:
also make sure `discussPublicView` exists before following relations on it